wiki/Do_you_really_want_to_ask_peppy/pl.md

```diff
@@ -0,0 +1,15 @@
+# Czy naprawdę chcesz skontaktować się z peppym?
+
+Cześć. Ja ([peppy](/users/2)) rozumiem twoją chęć do prywatnego skontaktowania się ze mną, ale musisz najpierw zrozumieć, że to **nie** jest miejsce, gdzie możesz się mnię pytać o pomoc. Oddzielam pomoc od prywatnych wiadomości. Musisz także zrozumieć, że odpowiadanie na prywatne wiadomości na forum jest dla mnie trudne do nadążenia. W związku z tym możesz użyć jednej z poniższych opcji:
+
+## Pomoc techniczna
+
+Większość problemów technicznych można rozwiązać poprzez szukanie lub postowanie na [Forum pomocy technicznej](/community/forums/5). Jest tam wiele innych osób, które mogą ci pomóć, więc znajdziesz tam rozwiązanie szybciej, niż kontaktując się ze mną.
+
+## Generalna pomoc
+
+Dla problemów z kontem, zapłatą, działaniami związanymi z banem, pytań związanych z turniejami i **wszystkiego związanego z potrzebą pomocy związanej z osu!** wyślij e-maila na adres [accounts@ppy.sh](mailto:accounts@ppy.sh) (pytania dotyczące kont) lub [osu@ppy.sh](mailto:osu@ppy.sh) (wszystkie inne). Jeśli jakiś z członków zespołu pomocy jest dostępny, możesz użyć małego przycisku ze znakiem zapytania, który znajduje się w prawym dolnym rogu strony, którą oglądasz.
```

Comment:
Dla problemów związanych z kontem, opłatami, korespondencji dotyczącej blokad konta, pytań odnośnie turniejów oraz **wszystkich pozostałych kwestii wymagających wsparcia dla osu!** wyślij e-maila na adres [accounts@ppy.sh](mailto:accounts@ppy.sh) (pytania dotyczące kont) lub [osu@ppy.sh](mailto:osu@ppy.sh) (pozostałe). Jeśli któryś z członków zespołu pomocy technicznej jest dostępny, możesz użyć małego przycisku ze znakiem zapytania, który pojawi się w prawym dolnym rogu.

^ to ostatnie zdanie chyba będzie miało najwięcej sensu w tej formie, chyba że ktoś ma lepszy pomysł

Reply:
"wszystkich pozostałych kwestii wymagających wsparcia dla osu!" to zdanie mi się jakoś nie klei, narazie może zamienię na "wszystkich pozostałych, związanych z osu!, kwestii wymagających wsparcia"
resztę zmieniłem

Reply:
To wtrącenie też takie średniawe.

W sumie można poprzestać na samym „wszystkich pozostałych kwestii”, bo w sumie zdanie jest już wystarczająco długie, a z kontekstu jest jasne, że chodzi o pomoc z osu!.